trestle/core/control_io.py

```diff
@@ -1122,7 +1122,7 @@ def get_control_param_dict(control: cat.Control, values_only: bool) -> Dict[str,
         return param_dict
 
     @staticmethod
-    def read_control(control_path: pathlib.Path) -> cat.Control:
+    def read_control(control_path: pathlib.Path) -> Tuple[cat.Control, str]:
         """Read the control markdown file."""
```

Comment:
Change it to 'read the control markdown file and group title'?

Reply:
The existing tests were extended to do additional testing of group title - which otherwise had not been checked.  It was confused with group_id in the code.  The issue with set-parameters was also being tested but with the wrong logic.  The existing test is now doing the right thing.  I also clarified the doc string you pointed out and provided more comments regarding group title.  Also simplified a function with awkward naming that parses the group title.